src/AppBundle/Command/AddUserCommand.php

```diff
@@ -222,7 +222,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
     }
 
-    private function passwordValidator($plainPassword)
+    /**
+     * This internal method should be private, but PHP requires callbacks to be declared public.
```

Comment:
I don't think it should be private, as you pass it to something else.

If you want to make it private, use a closure as actual validator though